modules/social_features/social_event/modules/social_event_an_enroll/src/Form/EventAnEnrollForm.php

```diff
@@ -57,6 +59,32 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#description' => $this->t('Enter your email, so we can send you event updates.'),
     ];
 
+    if ($this->moduleHandler->moduleExists('data_policy')) {
+      $form['#attached']['library'][] = 'core/drupal.dialog.ajax';
+
+      $link = Link::createFromRoute($this->t('data policy'), 'data_policy.data_policy', [], [
+        'attributes' => [
+          'class' => ['use-ajax'],
+          'data-dialog-type' => 'modal',
+          'data-dialog-options' => Json::encode([
+            'title' => t('Data policy'),
+            'width' => 700,
+            'height' => 700,
+          ]),
+        ],
+      ]);
+
+      $enforce_consent = !empty($this->configFactory->get('data_policy.data_policy')->get('enforce_consent'));
+
+      $form['data_policy'] = [
+        '#type' => 'checkbox',
+        '#title' => $this->t('I agree with the @url', [
+          '@url' => $link->toString(),
+        ]),
+        '#required' => $enforce_consent && $this->currentUser->isAnonymous(),
```

Comment:
@ribel Isn't the user always anonymous here? : )

Reply:
You are right, I will remove it.